### PR TITLE
Finish edit and destroy trainee

### PR DESCRIPTION
**Pull request by luongvietdung**

### CODE REVIEW

**Comment by vinhnd2010** [comment]
OK

File: app/views/supervisor/users/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<h1>
+  <%= t(".all_trainees") %>
```
Comment (hellovietnam93) [inline app/views/supervisor/users/index.html.erb]: should use t ".all_trainees" instead of

File: app/views/supervisor/users/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        </td>
+        <td>
+          <%= link_to t("application.buttons.remove"), supervisor_user_path(user),
+            method: :delete, data: {confirm: "Are you sure?"}, class: "btn btn-danger" %>
+        </td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
+<%= will_paginate @users %>
```
Comment (ThuBM) [inline app/views/supervisor/users/index.html.erb]: remove @users

File: app/controllers/supervisor/users_controller.rb [hunk 3]
```diff
@@ -1,20 +1,61 @@
...
   def user_params
     params.require(:user).permit :name, :email, :role
   end
+
+  def load_user
+    @user = User.find params[:id]
+  end
+
+  def authorize_supervisor
+    unless  current_user.supervisor?
```
Comment (ThuBM) [inline app/controllers/supervisor/users_controller.rb]: remove 1 space after unless

**Comment by ThuBM** [comment]
OK

**Pull request merged**